.github/workflows/helm.sh

```diff
@@ -1,45 +1,31 @@
 #!/bin/sh
 set -eu
 
-SUB_PROJECTS_DIRS="./hypertrace-alert-engine ./metric-anomaly-task-manager"
-
 subcommand=$1; shift
```

Comment:
Why is there a script here? Please use the shared github actions:

Example of packing and publishing:
https://github.com/hypertrace/config-service/blob/777df107b0141eb3368516c6527b4b846ea2d539/.github/workflows/publish.yml#L58-L62

Example of validating:
https://github.com/hypertrace/config-service/blob/777df107b0141eb3368516c6527b4b846ea2d539/.github/workflows/pr-test.yml#L68-L69

Reply:
Done